users/models.py

```diff
@@ -32,7 +62,10 @@ class User(AbstractUser):
     email = models.EmailField(unique=True)
     date_of_birth = models.DateField(blank=True, null=True)
     study = models.CharField(default="", blank=True, max_length=100)
-    profile_image = models.FileField(upload_to='profiles/', null=True)
+    profile_image = models.FileField(upload_to='profiles/', null=True, blank=True)
+    serious_profile_image = models.FileField(upload_to='profiles', null=True, blank=True)
+    in_relationship = models.BooleanField(default=False)
```

Comment:
Maybe put this as a `NullBooleanField`? I can see some cases where someone wouldn't want their relationship status shown

Reply:
This could also be solved by adding a preference to hide/show relationship status, but then we would need a separate field/mechanism to handle this.